package.json

```diff
@@ -11,7 +11,7 @@
         "seed": "yarn build-ts && yarn build-nest && yarn copy-html-files && node dist/server/scripts/seed.js -c config.seed.yaml",
         "seed:ci": "yarn build-ts && yarn build-nest && yarn copy-html-files && node dist/server/scripts/seed.js -c config.seed.test.ci.yaml",
         "dev": "yarn copy-static-assets && concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" -c \"cyan.bold,green.bold\" \"yarn watch-ts\" \"yarn watch-node\"",
-        "test": "env-cmd --silent jest --detectOpenHandles --forceExit",
+        "test": "env-cmd --silent node --max-old-space-size=8192 ./node_modules/.bin/jest --detectOpenHandles --forceExit",
```

Comment:
issue: restore the previous command because the memory used is too much for the CI environment and is probably not needed.